app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="user_container">
+  <div class="image"></div>
+  <div class="user">
+    <h2><%= link_to user.name, user %></h2>
+  <div class="post_counter">Number of Posts: <%= user.post_counter %></div>
+  </div>
+</div>
```

Comment:
- [ ] Please also display the user profile photo for the user as on the wireframe.